containers/ncbi_datasets_v16.10.0.def

```diff
@@ -0,0 +1,55 @@
+Bootstrap: docker
```

Comment:
Can the license header be added in these types of files?

Reply:
Tested and yes it appears license headers can be added.